custom/icds_reports/migrations/0104_citus_composite_key.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from django.apps import apps
+from django.db import connections, migrations
+from django.conf import settings
+
+from corehq.sql_db.routers import db_for_read_write
+
+
+def _citus_composite_key_sql(model_cls):
+    connection = connections[db_for_read_write(model_cls)]
+    with connection.schema_editor() as schema_editor:
+        pkey_name = '{}_pkey'.format(model_cls._meta.db_table)
+        fields = list(model_cls._meta.unique_together)[0]
```

Comment:
is this how it gets around not creating the indexes when you run `makemigrations`? I think this would break if someone attempted to add a `unique_together` constraint before the currently defined one. is that correct? 

it seems pretty hacky, but probably low likelihood to ever be hit. Could you add a comment about how this is working in case someone does run into in the future?

Reply:
(if I understand the question correctly) No this isn't. Because `state_operations` are being passed, `makemigrations` won't create a new migration file. But, if someone modifies `unique_together` and does a `makemigrations`, it _will_ create the necessary migration file that creates the indices.